src/components/Event.js

```diff
@@ -3,14 +3,16 @@ import React from 'react';
 export const Event = event => {
   return(
     <li className='item-card'>
-      <a href={event.href}>
+      <a href={event.url}>
         <div className='item-container'>
-          <img src={event.img}/>
+{/*          <img src={event.logo.url}/>*/}
```

Comment:
@epq, it is a placeholder I put until I can have the data on the Event component!